core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1603,7 +1602,8 @@ public Label getLabel(String expr) {
      * Returns the label atom of the given name.
      * @return non-null iff name is non-null
      */
-    public @Nullable LabelAtom getLabelAtom(@CheckForNull String name) {
+    @CheckForNull
+    public LabelAtom getLabelAtom(@CheckForNull String name) {
```

Comment:
Well, FB can warn me during build time. I'm getting all this weird cases only after user reported them (in this time first), so it should be better to indicate nullness for checker. End user can suppress FB warning after it will be sure that non-null is passed as argument. But ideally another method should be introduced also.